pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>idm-user-management</artifactId>
-  <version>0.2.3-SNAPSHOT</version>
+  <version>0.2.3</version>
```

Comment:
Good catch. Although, that commit was unintentional. I only meant it for local maven installation so client pom dependency needn't change. I'll correct it.